### ISSUE TITLE
Dynamic import does translate correctly in some cases (0.18.1)

### ISSUE THREAD

**Issue by charlag** [PepsRyuu/nollup#209]
Hi again!

We have such a piece of code generated now:

```js
__e__('createDropdown', function () { return createDropdown });function createAsyncDropdown(lazyButtons, width = 200) {
  // not all browsers have the actual button as e.currentTarget, but all of them send it as a second argument (see https://github.com/tutao/tutanota/issues/1110)
  return (e, dom) => {
    let originalButtons = lazyButtons();
    let buttons = originalButtons;
    // If the promise is pending and does not resolve in 100ms, show progress dialog
    if (originalButtons.isPending()) {
      buttons = Promise.race([
      originalButtons,
      Promise.all([
      Promise.delay(100),
      require.dynamic('/home/ivk/dev/repositories/tutanota-3/src/gui/dialogs/ProgressDialog.js')]).
      then(([_, module]) => {
        if (originalButtons.isPending()) {
          return module.showProgressDialog("loading_msg", originalButtons);
        } else {
          return originalButtons;
        }
      })]);


    }
    buttons.then((buttons) => {
      let dropdown = new DropdownN(() => buttons, width);
      dropdown.setOrigin(dom.getBoundingClientRect());
      modal.displayUnique(dropdown, false);
    });
  };
};
```

It fails on `require.dynamic()` because `require()` is not defined (it's not in the scope I think). Maybe another hoisting issue?
Thanks!

**Comment by PepsRyuu** [comment]
Nice catch! That definitely is an oversight for the reason you stated. Should be an easy fix, will have a look!

**Comment by PepsRyuu** [comment]
I seem to have having lots of difficulty getting Tutanota to work properly and can't easily debug it. I'd appreciate it if you could try the latest version of the master branch and let me know if it solves your problem.

**Comment by charlag** [comment]
Just tried it out on master, it seems to work, thanks!

Generally it's enough to 
```bash
npm ci
node make prod
python3 -m http.server
firefox http://localhost:9000/build
```

**Comment by PepsRyuu** [comment]
Great! Will release later today. 🙂

**Comment by PepsRyuu** [comment]
Released in ```0.18.2```!
